### PR TITLE
chore(react-chart): add api reference

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: packages/dx-react-chart/docs/reference/argument-axis.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# ArgumentAxis Plugin Reference
```
Comment (SergeyAlexeev) [inline packages/dx-react-chart/docs/reference/argument-axis.md]: What about the `Plugin Components` section?

Comment (Krijovnick) [reply packages/dx-react-chart/docs/reference/argument-axis.md]: yes, I'll add it now

File: packages/dx-react-chart/docs/reference/legend.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import { Legend } from '@devexpress/dx-react-chart-material-ui';
+// import { Legend } from '@devexpress/dx-react-chart-bootstrap4';
+```
+## User Reference
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+markerComponent | ComponentType&lt;Legend.MarkerProps&gt; | | A component that renders the marker.
```
Comment (SergeyAlexeev) [inline packages/dx-react-chart/docs/reference/legend.md]: Where is the `MarkerProps` described?

Comment (Krijovnick) [reply packages/dx-react-chart/docs/reference/legend.md]: There are no specific props expect className in the markerComponent. So there is nothing to describe.

Comment (SergeyAlexeev) [reply packages/dx-react-chart/docs/reference/legend.md]: OK. If we don't have an interface for this ComponentType, can we use `ComponentType<object>` type? We already use such a type for the [Table](https://devexpress.github.io/devextreme-reactive/react/grid/docs/reference/table/) plugin.

File: packages/dx-react-chart/docs/reference/argument-axis.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## User Reference
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+tickSize | number | 5 | Length of a tick.
+position | 'bottom' &#124; 'top' | 'bottom' | Axis position.
+name | string | | Axis name.
+indentFromAxis | number | 10 | Indent from axis
```
Comment (MaximKudriavtsev) [inline packages/dx-react-chart/docs/reference/argument-axis.md]: Missing dot at end of the sentence.

File: packages/dx-react-chart/docs/reference/area-series.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+Name | Type | Default | Description
+-----|------|---------|------------
+name | string | | A series name.
+valueField | string | | Data field provides values for series points.
+argumentField | string | | Data field provides arguments for series points.
+axisName | string | | Axis to which the series is bonded.
+stack | string | | Specifies which stack the series should belongs to.
+point | { size : number } | point: { size: 7 } | Specifies point options.
+seriesComponent | ComponentType&lt;[AreaSeries.SeriesProps](#areaseriesseriesprops)&gt; | | A component that renders the series.
+pointComponent | ComponentType&lt;[AreaSeries.PointProps](#areaseriespointprops)&gt; | | A component that renders the points.
```
[the remark this answers is not in the document]
Comment (Krijovnick) [reply packages/dx-react-chart/docs/reference/area-series.md]: These options do not make sense for the area series.

**Pull request merged**